app/ui/src/app/app.component.ts

```diff
@@ -12,9 +12,10 @@ import { saveAs } from 'file-saver';
 import { Restangular } from 'ngx-restangular';
 import { Notification, NotificationEvent } from 'patternfly-ng';
 import { Observable } from 'rxjs/Observable';
-import { ModalService } from '@syndesis/ui/common/modal/modal.service';
-import { NavigationService } from '@syndesis/ui/common/navigation.service';
-import { UserService, User } from '@syndesis/ui/platform';
+import { Store } from '@ngrx/store';
+import { UserService, User, PlatformActions, PlatformState } from '@syndesis/ui/platform';
+import { ModalService } from './common/modal/modal.service';
```

Comment:
Can we continue using the "@syndesis/ui/" path alias here?

Reply:
Root elements (app.component.ts, app.module.ts, app.etc.ts) usually refer to the folders themselves instead. It’s a good way to inform hierarchies visually.

Note to self: include modal and all its code entities as part of the main common barrel.